tests/integration/studies_blueprint/test_get_studies.py

```diff
@@ -1473,3 +1516,21 @@ def test_get_studies__invalid_parameters(
         assert res.status_code == INVALID_PARAMS_STATUS_CODE, res.json()
         description = res.json()["description"]
         assert re.search(r"could not be parsed to a boolean", description), f"{description=}"
+
+
+def test_studies_counting(client: TestClient, admin_access_token: str, user_access_token: str) -> None:
+    # test admin studies count
+    res = client.get(STUDIES_URL, headers={"Authorization": f"Bearer {admin_access_token}"})
+    assert res.status_code == 200, res.json()
+    expected_studies_count = len(res.json())
+    res = client.get(STUDIES_URL + "/count", headers={"Authorization": f"Bearer {admin_access_token}"})
+    assert res.status_code == 200, res.json()
+    assert res.json() == expected_studies_count
+
+    # test user studies count
```

Comment:
Do a loop around `admin_access_token` and `user_access_token` as it's exactly the same code